### PR TITLE
[4.0] Added a test to JHighlighter AutoUnhighlighter

### PR DESCRIPTION
**Pull request by yasirunilan**
Pull Request for Issue # .

### Summary of Changes
Added a javascript test to AutoUnhighlight option of JHighlighter library

### CODE REVIEW

File: tests/javascript/highlighter/spec.js [hunk 1]
```diff
@@ -95,4 +95,18 @@ define(['jquery', 'testsRoot/highlighter/spec-setup', 'jasmineJquery'], function
...
 		});
 	});
+
+	describe('Highlight single text with AutoUnhighlight false', function () {
+		beforeAll(function () {
+			highlighterAutohighlight.highlight(["text"]);
+		});
+
+		it('Should highlight sample text at depth 1', function () {
+			expect($('#text-depth-1').html()).toContain('class=\"highlight\"');
```
Comment (Ruchiranga) [inline tests/javascript/highlighter/spec.js]: The documentation here might be helpful for you to know.
https://github.com/velesin/jasmine-jquery

Comment (yasirunilan) [inline tests/javascript/highlighter/spec.js]: Thanks @Ruchiranga

**Comment by yasirunilan** [comment]
Can be merged??

**Comment by wilsonge** [comment]
@dgt41 can you review please

**Comment by ghost** [comment]
please reassign Milestone 4.0, thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14953">issues.joomla.org/tracker/joomla-cms/14953</a>.</sub>

**Comment by laoneo** [comment]
@dgrammatiko you are fine with this one?

**Comment by dgrammatiko** [comment]
aren’t the tests already moved in another repo?

**Comment by brianteeman** [comment]
@rdeutz how should we handle this now that all the tests have been moved

**Comment by brianteeman** [comment]
@puneet0191 as @rdeutz isnt replying can you please advise on what should be done here

**Comment by puneet0191** [comment]
@brianteeman yes, this one has to be merged into https://github.com/joomla/test-javascript 
@yasirunilan  can you please move the tests to the repo I pointed above, also fix the conflicts.

**Comment by brianteeman** [comment]
@puneet0191 Thanks for updating. Closing it here

**Pull request closed without merging**